src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -262,7 +262,18 @@ class ThumbnailsPanel extends React.PureComponent {
   };
 
   onPageNumberUpdated = pageNumber => {
-    this.listRef.current?.scrollToRow(pageNumber - 1);
+    const {
+      width,
+    } = this.state;
+    const numberOfColumns = this.getNumberOfColumns(width);
+    // from doing some drawings, page numbers are always ordered from left to right in asc order
+    new Array(numberOfColumns).fill().forEach((columnIndex) => {
+      if (pageNumber % (columnIndex + 1) === 0) {
```

Comment:
It seems that this `if` check will always be false because `columnIndex` is always `undefined`.
Can you explain a little bit more about `from doing some drawings?`, I don't think I see any `drawings` in the comments. Maybe I'm misunderstanding something?

Also I feel like the logic can be simplied to
```js
const numberOfColumns = this.getNumberOfColumns(this.state.width);
const pageIndex = pageNumber - 1;
this.listRef.current?.scrollToRow(Math.floor(pageIndex / numberOfColumns));
```

Reply:
```Can you explain a little bit more about from doing some drawings?```
I drew a few examples in my notebook before I came up with a formula.

But i think your formula is way easier to read compared to mines. so yes, i think we can use your suggestion.

Reply:
i have removed the comment also as it might cause some confusions if you got confused.